beacon-chain/powchain/service.go

```diff
@@ -245,6 +247,11 @@ func (s *Service) Status() error {
 	return nil
 }
 
+// ETH1Connection checks if the beacon node is connected to a ETH1 Node.
+func (s *Service) ETH1Connection() bool {
```

Comment:
Can this be a boolean type name?

```suggestion
func (s *Service) HasETH1Connection() bool {
```